bmrg-flow/templates/deployment-service.yaml

```diff
@@ -123,7 +143,13 @@ spec:
       {{- if and (eq $k "controller") (eq $values.workers.logging.type "elastic" ) }}
       - name: secret-volume
         secret:
-          secretName: {{ $values.workers.logging.secret }}
+          secretName: {{ include "bmrg.name" (dict "context" $ "tier" "certs" ) }}
+      {{- else }}
+      {{- if $.Values.database.mongodb.tls.enable }}
+      - name: secret-volume
+        secret:
+          secretName: {{ include "bmrg.name" (dict "context" $ "tier" "certs" ) }}
+      {{- end }}
```

Comment:
Naming them both secret-volume is potentially going to lead to a collision if both MongoDB TLS and Elastic are enabled at the same time